index.js

```diff
@@ -11,9 +11,13 @@ function resolve(targetUrl, source) {
 
   var filePath = path.resolve(packageRoot, 'node_modules', targetUrl);
 
-  return fs.existsSync(path.dirname(filePath))
-    ? filePath
-    : resolve(targetUrl, path.dirname(packageRoot));
+  if (fsUtil.isDirectory(filePath) && fsUtil.existsSync(filePath)) {
+    return path.resolve(filePath, 'index');
```

Comment:
It looks like we have lost the test case for this functionality since the last change...

Reply:
we have tests for it, the coverage is 100%, but I'll add 2 explicit tests 2 sec